frontend/lib/package.json

```diff
@@ -7,11 +7,11 @@
     "types": "dist/index.d.ts",
     "sideEffects": false,
     "scripts": {
-        "build": "yarn build:src && yarn build:vendor && yarn build:assets && yarn build:proto && yarn build:lib-type",
-        "build:lib-type": "tsc -P tsconfig.json && tsc-alias -p tsconfig.json",
-        "build:tsc": "tsc -P tsconfig.json",
-        "build:tsc-alias": "tsc-alias -p tsconfig.json",
-        "build:src": "env NODE_ENV=\"development\" ./node_modules/.bin/babel --root-mode=upward --out-dir dist --source-maps --extensions \".js,.mjs,.jsx,.ts,.tsx\" src",
+        "build": "yarn build:src && yarn build:vendor && yarn build:assets && yarn build:proto && yarn build:lib-dev",
```

Comment:
Dev and Prod versions are necessary because if we build `@streamlit/lib` with relative pathing, then hot reloading will not work as the dist folder as our webpack dev server will not be pointing to the right location as relative pathing will point to the dist folder rather than the source files.